examples/plugin/errormessage/README.md

```
@@ -0,0 +1,89 @@
+
+## Usage
+
+```bash
+cd examples/plugin/errormessage
+go run .
+```
+
+You will see two blocks in the output. The first block is persisted **without**
+the plugin, so the default fallback message is stored:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)